web/packages/teleterm/src/ui/DocumentConnectMyComputer/DocumentConnectMyComputerSetup.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2023 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { useState } from 'react';
+import { Box, ButtonPrimary, Text } from 'design';
+
+import * as types from 'teleterm/ui/services/workspacesService';
+import Document from 'teleterm/ui/Document';
+
+interface DocumentConnectMyComputerSetupProps {
+  visible: boolean;
+  doc: types.DocumentConnectMyComputerSetup;
+}
+
+export function DocumentConnectMyComputerSetup(
+  props: DocumentConnectMyComputerSetupProps
+) {
+  const [step, setStep] =
+    useState<'information' | 'agent-setup'>('information');
+
+  return (
+    <Document visible={props.visible}>
+      <Box maxWidth="590px" mx="auto" mt="4" px="5" width="100%">
+        <Text typography="h3" mb="4">
+          Connect My Computer
+        </Text>
+        {step === 'information' && (
+          <Information onSetUpAgentClick={() => setStep('agent-setup')} />
+        )}
+        {step === 'agent-setup' && <AgentSetup />}
+      </Box>
+    </Document>
+  );
+}
+
+function Information(props: { onSetUpAgentClick(): void }) {
+  return (
+    <>
+      {/* TODO(gzdunek): change the temporary copy */}
```

Comment:
I added this to the tracking issue, let's keep adding stuff there on top of adding TODO comments, it's easier to keep track of it this way.